docs/config.rst

```diff
@@ -1,11 +1,502 @@
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+There are four steps to this flow. The first three steps are themselves flows, and the last step is a task.
+
+All **non-negative numbers and decimals** are valid as step numbers in a flow.
+This makes it easy to add steps before, in-between, or after existing flow steps.
+If we wanted to add a step at the beginning of the dev org flow, valid step numbers would include any number greater than or equal to zero and less than 1.
+Example values would include: 0, 0.3, and 0.89334.
+All of these would cause the step to execute before step 1 in the ``dev_org`` flow.
+
+If you wanted to add a step between steps 2 and 3, then a step number of 2.5 could be utilized.
+
+If you wanted to add a step number that runs after all steps in the flow, then any step number greater than 4 could be utilized.
+
+
+
+Skip a Flow Step
+^^^^^^^^^^^^^^^^
+To skip a flow step, set the desired step number to a task with the value of ``None``.
+The following would skip the 4th step from the ``dev_org`` flow.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: None
+
+When CumulusCI detects a task with this value, it is skipped:
+
+.. image:: images/skipping_task.png
+
+
+
+Replace a Flow Step
+^^^^^^^^^^^^^^^^^^^
+To replace a flow step, simply name the task or flow you wish to run in place of the current step.
+The following would replace the fourth step of the ``dev_org`` flow with a custom task that loads data into a dev environment.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: load_data_dev
+
+You can swap two steps in a flow by replacing one with the other. 
+
+
+
+
+
+
+Configuring Options on Tasks When Running a Subflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TODO
+
+
+
+Using ``when`` Clauses
+^^^^^^^^^^^^^^^^^^^^^^
+TODO
+
+
+Org Configurations
+-------------------
+This section shows you the values that are overridable in an org configuration via the `cumulusci.yml` file.
+If you're looking for more information using scratch orgs with CumulusCI, see `Scratch Org Environments`_. 
+
+Override a Default Values
+^^^^^^^^^^^^^^^^^^^^^^^^^
+These overrides pertain only to scratch orgs.
+You can override the following values for org:
+
+* ``days`` - integer - Number of days for the scratch org to persist.
+* ``namespaced`` - boolean - Whether or not the scratch org is a `namespaced org <TODO sf link?>`_.
+* ``config_file`` - string - Path to the org definition file to use when building the scratch org.
+
+To override a value you can use the following syntax
+
+.. code-block:: yaml
+    
+    orgs:
+        org_name:
+            key: value 
```

Comment:
If I'm not mistaken, there's another key `scratch:` which needs to be included here at a new level of indentation in between `orgs:` and the config name.